data/swapnul.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Swapnil Srivastava",
+  "type": "personal",
+  "bio": "Quality Engineer | Java | Learner",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Swapnil__404",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/swapnul",
+      "icon": "FaTwitter"
```

You might want to use the GitHub icon here.
```suggestion
      "icon": "FaGithub"
```